### ISSUE TITLE
Can't Remove keybinding

### ISSUE THREAD

**Issue by Juli-CVidal** [microsoft/vscode#179891]
Type: <b>Bug</b>

I accidentaly created a new shortcut via "Keyboard Shortcuts" and mapped the letter "a" without when condition, now everytime I press "a", it executes explorer.newFile.

I've tried everything, including reinstalling. 

VS Code version: Code 1.77.3 (704ed70d4fd1c6bd6342c436f1ede30d1cff4710, 2023-04-12T09:16:02.548Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 2400G with Radeon Vega Graphics     (8 x 3593)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|5.93GB (0.83GB free)|
|Process Argv|--crash-reporter-id 59535d0b-6daf-4c0c-be54-6d88651acd42|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (24)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|3.0.2
bootstrap5-vscode|Anb|0.4.2
html-end-tag-labels|ant|0.11.0
markdown-preview-github-styles|bie|1.0.1
path-intellisense|chr|2.8.4
vscode-eslint|dba|2.4.0
es7-react-js-snippets|dsz|4.4.3
gitlens|eam|13.5.0
prettier-vscode|esb|9.10.4
auto-close-tag|for|0.5.14
auto-rename-tag|for|0.1.10
isort|ms-|2022.8.0
python|ms-|2023.6.0
vscode-pylance|ms-|2023.4.10
jupyter|ms-|2023.3.1201040234
jupyter-keymap|ms-|1.1.0
jupyter-renderers|ms-|1.0.15
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
material-icon-theme|PKi|4.26.0
LiveServer|rit|5.7.9
sublime-mavensmate-monokai-apex|Sub|0.1.6
errorlens|use|3.9.0
JavaScriptSnippets|xab|1.8.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vscaac:30438847
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
nodejswelcome1:30587005
3biah626:30602489
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
a9j8j154:30646983
showlangstatbar:30672706
vsctsb:30705553
functionswalk:30687959
pythonms35:30701012
pythonfmttextcf:30710682

```

</details>

**Comment by sandy081** [comment]
Have you tried **Remove Keybinding** action from the context menu?

<img width="739" alt="image" src="https://user-images.githubusercontent.com/10746682/232554470-7249da5d-40e9-4435-b911-a6937d030678.png">

**Comment by Juli-CVidal** [comment]
Yes, I have tried several times. If I generate another assignment and delete it, it works, but not for the first one. 
The solution I found was to uninstall VsCode and delete its temporary folder.

**Comment by sandy081** [comment]
Can you please share your `keybindings.json` file contents. You can open the file using the command `F1 > Open Keyboard Shortcuts (JSON)`
